src/nvim/version.c

```diff
@@ -80,8 +80,8 @@ static int included_patches[] = {
   784,
   //783,
   //782,
-  //781,
-  //780,
+  781,
+  //780 NA
```

Comment:
In another PR @Pyrohh suggested to make an own commit for this, like you did before. I'll wait until I have at least 10 of these `NA` patches, before making a PR for it, though.